doc/news/changes.h

```diff
@@ -63,6 +63,12 @@ inconvenience this causes.
 
 
 <ol> 
+ <li> Fixed: The function make_trilinos_map() now works if some processors have 
```

Comment:
Can you qualify the function name with the class name so that doxygen automatically links to it?